packages/contracts-ethers/npm/index.ts

```diff
@@ -1,5 +1,2 @@
 // TODO
```

Comment:
Needed?

```suggestion
// TODO
```

Reply:
not sure what you mean, I just removed the imports that were not resolving

Reply:
Do we need to have a `// TODO` for the template?